server/.env.example

```
@@ -0,0 +1,34 @@
+# PostgreSQL connection URL
+# Example: postgresql://USER:PASSWORD@HOST:PORT/DATABASE?schema=public
+DATABASE_URL="postgresql://postgres:password@localhost:5432/chat_lite_db?schema=public"
+
+# A long, random, secret string for signing JWT tokens
+JWT_SECRET="your_super_secret_jwt_string_at_least_32_characters_long"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)